### PR TITLE
fixed rule api's in auth0

### PR DESCRIPTION
**Pull request by donaldpipowitch**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://auth0.github.io/node-auth0/module-management.ManagementClient.html, https://auth0.com/docs/api/management/v2
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

---

Note that it looks like the official docs had an error, too. I created a PR there as well https://github.com/auth0/node-auth0/pull/214.

### CODE REVIEW

File: types/auth0/index.d.ts [hunk 1]
```diff
@@ -377,7 +404,7 @@ export class ManagementClient {
   getUser(params: ObjectWithId, cb?: (err: Error, user: User) => void): void;
 
   createUser(data: CreateUserData): Promise<User>;
-  createUser(data: CreateUserData, cb: (err: Error, data: User) => void): void;
+  createUser(data: CreateUserData, cb: (err: Error, user: User) => void): void;
```
Comment (wbhob) [inline types/auth0/index.d.ts]: Thank you for this fix!

**Review by wbhob — APPROVED** [review]
Thank you for clearing up these changes! This is a good PR.

**Comment by wbhob** [comment]
@donaldpipowitch this is worthy of a patch version increment, can you add that in another commit (add to file header and package.json)
?

**Comment by donaldpipowitch** [comment]
@wbhob Thank you for review! It looks like this project has no `package.json`, is this correct? With file header you're referring to this line? https://github.com/donaldpipowitch/DefinitelyTyped/blob/e72ca2e160cbfde9a52a7c3a84d47be0cc660651/types/auth0/index.d.ts#L1

It looks like this version doesn't include a patch version (just major.minor). Should I change this line to `
// Type definitions for auth0 2.4`? Thank you.

**Comment by wbhob** [comment]
That's right @donaldpipowitch

**Comment by donaldpipowitch** [comment]
Pushed :)

**Comment by wbhob** [comment]
I approve! Thanks

**Comment by donaldpipowitch** [comment]
Cool. Thank you.

**Pull request merged**